apps/ideas/src/components/modules/IdeaGeneratorButton/IdeaGeneratorButton.js

```diff
@@ -21,26 +21,37 @@ export default function IdeaGeneratorButton(props) {
   const openIdeaModal = async () => {
     setModalContent(
       <ModalWrapper>
-        <span style={{fontSize:"1.25em"}}>Loading...</span>
+        <span style={{ fontSize: '1.25em' }}>Loading...</span>
         <br />
         <br />
-        We are querying a "Large Language Model" to generate a new idea using AI. It may take up to 30 seconds!
+        We are querying a "Large Language Model" to generate a new idea using
+        AI. It may take up to 30 seconds!
         <br />
```

Comment:
Outside the scope of this story, but it would be nice to remove these `<br />` tags

https://stackoverflow.com/questions/1726073/is-it-sometimes-bad-to-use-br

Reply:
I was thinking the same thing. We have one other potential change to make to this modal. Originally, after an idea generated, the Generate Idea button would appear in the modal so that you could click to generate another idea. But since I changed the style of the Generate Idea button a lot, it was obtrusive and didn't look good. If we we want users to have the ability to generate another idea while the modal is still up, we could create a story for that and include removing these <br> tags as part of that story.